grades/models.py

```diff
@@ -22,6 +22,23 @@ class CourseRunGradingAlreadyCompleteError(Exception):
     """
 
 
+class FinalGradeQuerySet(models.QuerySet):
+    """
+    QuerySet class defining common query parameters for FinalGrade
+    """
+    def passed(self):  # pylint: disable=missing-docstring
+        return self.filter(passed=True)
+
+    def paid(self):  # pylint: disable=missing-docstring
+        return self.filter(course_run_paid_on_edx=True)
```

Comment:
this is conceptually wrong: if `course_run_paid_on_edx` is `False` it does not mean that it has not been paid.